index.html

```diff
@@ -0,0 +1,2199 @@
+            <article class="w-100 sub-container">
+              <div class="d-flex">
+                <div>
+                  <img class="sub-img w-100 " src="https://d.newsweek.com/en/full/1579878/april-17-24-2020-cover.jpg" alt="subscribe-img">
+                </div>
+                <div class="d-flex flex-column align-items-start pt-5 sub-paragraph">
+                  <h2 class="mag_link2 py-2">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h2>
+                  <p class="py-2">All Access + Weekly Delivery.
+                    Choose A Membership That's Perfect for You!</p>
+                  <div class="py-2">
+                      <button type="button" class="subscribe_btn">SUBSCRIBE ></button>
+                  </div>
+                </div>
+              </div>
+            </article>
+          </section>
+        </div>
+        <div class="col-md-12 col-lg-4 signup-wrapper">
+          <div class=" col-lg-4 d-flex align-items-center justify-content-start font-roboto">
+            <h2 class="font-weight-bold sub-title">NEWSLETTER</h2>
+          </div>
+          <article class="w-100  sign-up-container">
+            <div class="d-flex">
+              <div>
+                <img class="sign-up-img w-100" src="https://d.newsweek.com/en/full/1579878/april-17-24-2020-cover.jpg" alt="subscribe-img">
+              </div>
+              <div class="d-flex flex-column align-items-start sub-paragraph">
+                <h2 class="mag_link2 py-2">The Starting 5</h2>
+                <p class="py-2">See why nearly a quarter of a million subscribers
+                   begin their day with the Starting 5.
+                </p>
+                  <div class="py-2">
+                    <button type="button" class="subscribe_btn">SIGN UP ></button>
+                  </div>                
+              </div>
+            </div>
+          </article>
+
+        </div>
+    
+      </div> 
+    </main>
+
+    <footer>
+      <div class="d-flex flex-md-row align-items-md-end justify-content-between pb-3 pt-3 border-bottom">
+        <div class="d-flex flex-column flex-md-row align-items-start align-items-md-end">
+          <div class="pl-3">
+            <img class="footer-logo" src="./footer-logo.PNG" alt="footer Logo">
+          </div>
+          <div>
+            <small class="footer-copyright text-muted text-uppercase">© 2020 Newsweek</small>
+          </div>
+        </div>
+        <ul class="d-flex justify-content-around social-icons mb-0">
+          <li><a href="#"><i class="px-2 text-muted fab fa-facebook-square"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-twitter"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-instagram"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-tumblr"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-linkedin-in"></i></a></li>
+        </ul>
+      </div>
```

Comment:
Give more space in between the lines of the footer elements, they look cramped.
Also, check the way the elements rearrange when you change screen sizes.
![Screenshot from 2020-04-22 13-41-24](https://user-images.githubusercontent.com/51926937/80020638-fcd8fe80-849e-11ea-97e1-e77580aa8b8b.png)

Original:
![Screenshot from 2020-04-22 13-41-17](https://user-images.githubusercontent.com/51926937/80020676-07939380-849f-11ea-8e5f-61a76fd6a4d9.png)